Add missing cluster_manager_throttling nodes stats (#7241)
opensearch-project/documentation-website: _api-reference/nodes-apis/nodes-stats.md

indexing_pressure | Statistics about the node's indexing pressure.
shard_indexing_pressure | Statistics about shard indexing pressure.
search_backpressure | Statistics related to search backpressure.

> cluster_manager_throttling | Statistics related to throttled tasks on cluster manager node.

Reviewer: ```suggestion
cluster_manager_throttling | Statistics related to throttled tasks on the cluster manager node.
```

[indexing_pressure](#indexing_pressure) | Object | Statistics related to the node's indexing pressure.
[shard_indexing_pressure](#shard_indexing_pressure) | Object | Statistics related to indexing pressure at the shard level.
[search_backpressure]({{site.url}}{{site.baseurl}}/opensearch/search-backpressure#search-backpressure-stats-api) | Object | Statistics related to search backpressure.

> [cluster_manager_throttling](#cluster_manager_throttling) | Object | Statistics related to throttled tasks on cluster manager node.

Reviewer: ```suggestion
[cluster_manager_throttling](#cluster_manager_throttling) | Object | Statistics related to throttled tasks on the cluster manager node.
```

enabled | Boolean | Specifies whether the shard indexing pressure feature is turned on for the node.
enforced | Boolean | If true, the shard indexing pressure runs in enforced mode (there are rejections). If false, the shard indexing pressure runs in shadow mode (there are no rejections, but statistics are recorded and can be retrieved in the `total_rejections_breakup_shadow_mode` object). Only applicable if shard indexing pressure is enabled. 

### `cluster_manager_throttling`

> The `cluster_manager_throttling` object contains statistics about throttled tasks on the cluster manager node. It is populated only for actual cluster manager node.  

Reviewer: ```suggestion
The `cluster_manager_throttling` object contains statistics about throttled tasks on the cluster manager node. It is populated only for nodes that have a cluster manager role.  
```

Author: I think this has a bit different meaning. I will double-check but the original meaning was that the statistics will be provided only for the node that is currently elected as a cluster-manager. Not for all cluster-manager eligible nodes.

Reviewer: Thank you for clarifying. Let me update the suggestion.

Field | Field type | Description
:--- | :--- | :---
stats | Object | Statistics about throttled tasks on the cluster manager node.

> stats.total_throttled_tasks | Long | Total number of throttled tasks.

Reviewer: ```suggestion
stats.total_throttled_tasks | Long | The total number of throttled tasks.
```

> stats.throttled_tasks_per_task_type | Object | Statistics breakdown by individual task types. The object keys are individual types of tasks and their value represents the number of requests that were throttled.

Reviewer: ```suggestion
stats.throttled_tasks_per_task_type | Object | Statistics breakdown by individual task type specified as key-value pairs. The keys are individual task types and their values represent the number of requests that were throttled.
```

> stats.throttled_tasks_per_task_type | Object | Statistics breakdown by individual task type specified as key-value pairs. The keys are individual task types and their values represent the number of requests that were throttled.

Reviewer 2: ```suggestion
stats.throttled_tasks_per_task_type | Object | A breakdown of statistics by individual task type, specified as key-value pairs. The keys are individual task types, and their values represent the number of requests that were throttled.
```
